index.html

```diff
@@ -49,177 +51,197 @@ <h1 class="intro-title">I’am a Software Developer</h1>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
-            </div>
-        </div>
-        <button type="button" class="see-jobs-btn">See Project</button>
-        <div class="job-item-two">
-            <h1 class="job-post-title">Profesional Art Printing Data</h1>
-            <div class="job-post">
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            </div>
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
+            <div class="multi-post-inner">
+                <h1 class="post-title">Multi-Post Stories</h1>
+                <div class="post">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="tech-lang">
+                    <ul>
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li id="last-child">Ruby</li>
+                    </ul>
+                    <button type="button" id="post-btn">See Project</button>
```

Comment:
- [ ] Please ensure that the See project button directs a user to the live version of your project or your git hub repository. You can do this by using the `<a>` tag and styling it based on the Figma design.